utility make target no longer exists, but the man-page target depends on it

### Issue Summary
`make man-page` fails with the error:

```bash
make: *** No rule to make target 'utility', needed by 'man-page'.  Stop.
```

#### System Information
- **Operating System:** Arch Linux.
- **.NET / Mono Version:** Mono 5.20.1.
- **OpenRA Version:** commit number 25990, hash 83b4e44. 
- **Mod:** N/A.

#### Additional Information:
- Checkout clean copy of the repo at commit 83b4e44 on Arch Linux, with only stable repositories enabled.
- Run `make`.
- Run `make man-page` and voilà.

--- a/Makefile
+++ b/Makefile
@@ -212,9 +212,6 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-man-page: core
-	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
-
 install: dependencies core install-core
 
 install-linux-shortcuts: install-linux-scripts install-linux-icons install-linux-desktop
@@ -304,9 +301,11 @@ install-linux-appdata:
 	@$(INSTALL_DATA) packaging/linux/openra-d2k.appdata.xml "$(DESTDIR)$(datadir)/appdata/"
 	@-$(RM) packaging/linux/openra-ra.appdata.xml packaging/linux/openra-cnc.appdata.xml packaging/linux/openra-d2k.appdata.xml
 
-install-man-page: man-page
+install-man-page:
 	@$(INSTALL_DIR) "$(DESTDIR)$(mandir)/man6/"
+	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
 	@$(INSTALL_DATA) openra.6 "$(DESTDIR)$(mandir)/man6/"
+	@-$(RM) openra.6
 
 install-linux-scripts:
 ifeq ($(DEBUG), $(filter $(DEBUG),false no n off 0))
@@ -394,4 +393,4 @@ help:
 
 .SUFFIXES:
 
-.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version man-page install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help
+.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help

--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,6 @@ INSTALL_DATA = $(INSTALL) -m644
 MSBUILD = msbuild -verbosity:m -nologo
 
 # program targets
-CORE = pdefault game utility server
 VERSION     = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
 
 # dependencies
@@ -213,7 +212,7 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-man-page: utility mods
+man-page: core
 	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
 
 install: dependencies core install-core
@@ -395,4 +394,4 @@ help:
 
 .SUFFIXES:
 
-.PHONY: core package all mods clean distclean dependencies version nunit
+.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version man-page install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help

